src/components/ReportsPage/AllScorecardsPage/AllScorecardsHeatmap.tsx

```diff
@@ -18,6 +18,9 @@ import { Progress, WarningPanel } from '@backstage/core-components';
 import { useCortexApi } from '../../../utils/hooks';
 import { GroupByOption } from '../../../api/types';
 import { AllScorecardsHeatmapTable } from '../HeatmapPage/Tables/AllScorecardHeatmapTable';
+import {StringIndexable} from "../HeatmapPage/HeatmapUtils";
```

Comment:
We really should ensure this happens during CI, but please run `yarn pretty-quick` 🙏